### DISCUSSION TITLE
Burn too dark/too light/random

### DISCUSSION THREAD

**Discussion by monk1974** [arkypita/LaserGRBL#1533] [Q&A]
I recently upgraded from a cheap diy laser setup to a Ortur Laser Master 2 20w.

Now I will be posting screenshots from lightburn but I get the same issues with lasergrbl!

My Uncle and Aunts dog recently passed away and I am trying to burn an image of there dog and its name. I have tried everything I’m comfortable with changing as far as settings but just can’t seem to figure it out.

My problem is I import the image,I have it on Jarvis and preview looks fine.
Though I have attempted to burn this image several times and messing with power settings,contrast,brightness,speed and no luck.
 
Its either the image is burned too dark and there is no lighter tones,or the image is too light and barely burns and ALSO several times the black lettering is just outlined and not burned at all. I have even tried calibration test burns and there is no gradients.

here is the image
![20210718_231531](https://user-images.githubusercontent.com/47600162/126234836-71921165-898b-43c3-a6bf-d1179883bc5b.jpg)
and here is my last attempt to burn
![20210718_230828](https://user-images.githubusercontent.com/47600162/126234933-4766a367-69f8-4ff4-a521-587cb5601b87.jpg)
 I am grateful for any all help.

**Comment by gmmanonymus111** [comment]
I have NEJE's 20W laser, and sadly, it can be used to either cut from wood, or engrave on harder materials (rocks or metals). It is too powerful to engrave a nice image with tones, especially on wood (If you look closer, you can see, that instead of color tones, you get deeper engravings. The stronger your laser power, the deeper the burn). I had to mount my older less powerful laser to be able to engrave a more "colorful" image on wood.
